cli/cluster/cmd/uri.go

```diff
@@ -144,17 +144,22 @@ func MakeEtcdOptsFromUriOpts(src UriOpts) cluster.EtcdOpts {
 
 // MakeConnectOptsFromUriOpts create Tarantool connect options from
 // URI options.
-func MakeConnectOptsFromUriOpts(src UriOpts) connector.ConnectOpts {
-	return connector.ConnectOpts{
-		Network:  connector.TCPNetwork,
-		Address:  src.Host,
-		Username: src.Username,
-		Password: src.Password,
-		Ssl: connector.SslOpts{
+func MakeConnectOptsFromUriOpts(src UriOpts) (tarantool.Opts, string) {
```

Comment:
```suggestion
func MakeConnectOptsFromUriOpts(src UriOpts) (string, tarantool.Opts, ) {
```

It is a bit confusing:
```Go
opts, addr := MakeConnectOptsFromUriOpts()
conn := tarantool.Connect(addr, opts)
```
It should be better to use:
```Go
addr, opts := MakeConnectOptsFromUriOpts()
conn := tarantool.Connect(addr, opts)
```

Reply:
Fixed